### ISSUE TITLE
.net6 templates work from local folder, but the nuget package does not

### ISSUE THREAD

**Issue by paulc1983** [sayedihashimi/template-sample#72]
Hi

I have created some templates and tried to publish them to the nuget gallery but they do not work. I can install and uninstall by using the windows file path, but when i auto generate the package, or use dotnet pack and then analyse the nupkg file it says no template found :

 templates analyze -p "E:\Repos\beis-register-your-interest-civilian-template - Copy\Beis.RegisterYourInterest\bin\Debug\Beis.RegisterYourInterest-Civilian.6.0.0-ci.0.nupkg"
analyzing...
verbose: extracting zip 'E:\Repos\beis-register-your-interest-civilian-template - Copy\Beis.RegisterYourInterest\bin\Debug\Beis.RegisterYourInterest-Civilian.6.0.0-ci.0.nupkg' to 'C:\Users\paulc\AppData\Local\templatereport\extracted\Beis.RegisterYourInterest-Civilian.6.0.0-ci.0.nupkg'
verbose: Deleting package cache directory because forceExtract is true, path = 'C:\Users\paulc\AppData\Local\templatereport\extracted\Beis.RegisterYourInterest-Civilian.6.0.0-ci.0.nupkg'
ERROR: No templates found under path 'C:\Users\paulc\AppData\Local\templatereport\extracted\Beis.RegisterYourInterest-Civilian.6.0.0-ci.0.nupkg'

here is the package that gets published in a yaml pipeline https://www.nuget.org/packages/Beis.RegisterYourInterest-Civilian/6.0.0-ci.0

it appears to have the assembly dll but looks pretty empty.  

can someone explain how to generate a valid nuget package for a .net6 mvc project?

thanks

paul

**Comment by baronfel** [comment]
Yep! There's a three-part series [here](https://learn.microsoft.com/en-us/dotnet/core/tutorials/cli-templates-create-template-package) that creates an item template, a project template, and finally packages them all together.
